### PR TITLE
add & add_json to return the ID of the posted message

### PR DESCRIPTION
**Pull request by timojaask**
Having the ID of the posted  message can be useful on the client side. For example when posting a message with a delay of 5 minutes. Then on the client side you notify the user that the message is pending. You set a timer to remove the pending notification in 5-6 minutes, unless this message arrives from the server when you do get messages. How do you know this message arrived? I think a good way would be to compare the ID, because comparing text can be misleading.

### CODE REVIEW

**Comment by sarukuku** [comment]
> @timojaask: I think a good way would be to compare the ID, because comparing text can me misleading.

As a general note we'll never want to compare content in this kind of case. It's not only misleading but it's a bad practice and an antipattern. For example one simple mutation in the content when saving to database with a different collation and the comparison fails. Always use something you know is unique for these kind of cases.

File: api.php [hunk 1]
```diff
@@ -31,10 +33,8 @@ public function add_json()
 		);
 
 		$this->db->insert('whispers', $data);
-
-		// TODO: would be nice if this function returned
-		// the ID of the new message.
-		echo "123";
+		// Return the ID of the newly inserted message
+		echo $this->db->insert_id();
 	}
```
Comment (timojaask) [inline api.php:39]: I like this empty line.

Comment (timojaask) [inline api.php:39]: Commenting on code like this is a cool feature! :ok_hand:
(sorry for useless notification, just testing things here)

Comment (sarukuku) [inline api.php:39]: GitHub <3

Comment (daitj) [inline api.php:39]: okay great :+1: 
should someone merge to master  :grin:

**Pull request merged**